lib/spack/llnl/util/link_tree.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, dst, src_a=None, src_b=None):
         self.src_b = src_b
 
 
-class SourceMergeVisitor(object):
+class SourceMergeVisitor(GenericVisitor):
```

Comment:
`visit_symlinked_file` should call `visit_file` here

Reply:
I'll make that update. Thanks.

Reply:
if there are no failing tests that'd be worrying ;p